src/Forms/UrlField.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+/**
+ * Text input field with validation for a url using filter_var($url, FILTER_VALIDATE_URL)
```

Comment:
```suggestion
 * Text input field with validation for a url
```
The PHPDoc shouldn't say what the internal implementation is since that could change at any time. Preferably say "according to RFC3986" instead, if you do want to give a clue about what sort of validation is done.

Reply:
Removed